sdk/appconfiguration/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/implementation/config/AppConfigurationBootstrapConfiguration.java

```diff
@@ -88,6 +93,12 @@ AppConfigurationReplicaClientFactory buildClientFactory(AppConfigurationReplicaC
         AppConfigurationProperties properties) {
         return new AppConfigurationReplicaClientFactory(clientBuilder, properties.getStores());
     }
+    
+    @Bean
+    @ConditionalOnMissingBean
+    AzureGlobalProperties azureGlobalProperties() {
+        return new AzureGlobalProperties();
+    }
```

Comment:
We should double check whether registering this bean will prevent registering one in the application context.

Reply:
I was trying to look at the other one's creation. It does have a different name, but I couldn't figure out if you had something stopping it if another instance of it already existed.